types/k6/test/experimental/streams.ts

```diff
@@ -0,0 +1,20 @@
+import {ReadableStream } from "k6/experimental/streams";
+
+// Note @oleiade: ReadableStreamDefaultReader's and ReadableStreamDefaultController's method
+// either take no arguments or take optional arguments only and thus do not need to be tested. 
```

Comment:
```suggestion
// ReadableStreamDefaultReader's and ReadableStreamDefaultController's method
// either take no arguments or take optional arguments only and thus do not need to be tested. 
```
Is it required?

Reply:
It is a practice that I don't like because it provides personalism to source code where knowledge sharing and interchangeability should be the goal.

In any case, it is a subjective opinion so feel free to keep it.